server/graphql/utils/request.ts

```diff
@@ -20,19 +22,60 @@ const getRequestsById = (requestIds, dataSources) => {
   return requestIds.map(id => dataSources.requests.getById(id))
 }
 
-const updateRequestHelper = (request, dataSources): Promise<ServerResponseInterface> => {
+const updateRequestHelper = async (request, dataSources): Promise<Document> => {
   if(!request.id) {
     throw new UserInputError('Missing argument value', { argumentName: 'id' })
   }
-  const requestTypeId = dataSources.requests.getById(request.id.toString()).requestType
-  return dataSources.requests.update(request)
-    .then(res => {
-      updateRequestTypeHelper({"id": requestTypeId}, dataSources)
-      return res
-    })
-    .catch(error => {
-      return error
-    })
+  const session = await mongoose.startSession()
+  try {
+    session.startTransaction()
+    const currentRequest = dataSources.requests.getById(request.id.toString())
```

Comment:
Ideally, you do not need to store old versions with transaction. Transaction is performed isolated from the live database and is only finished if committed and "undone" if aborted. So you shouldn't need current/old versions ideally

Reply:
This is just so we can retrieve the requestType field of the current request object. There isn't any old versions being stored here.

Reply:
Sorry misplaced the comment. Line below

Reply:
Are you referring to the old/new requestTypeId's? Those are for referring to the old and new versions of that field, since we need to modify the old/new requestTypes, not old/new in the sense of storing old versions. 
Maybe prev/new would be better wording?

Reply:
Okay I understand now. Makes sense!